base_layer/key_manager/src/cipher_seed.rs

```diff
@@ -130,19 +145,18 @@ impl CipherSeed {
 
         let passphrase = passphrase.unwrap_or_else(|| DEFAULT_CIPHER_SEED_PASSPHRASE.to_string());
 
-        // Construct HMAC and include the version and salt as Associated Data
-        let blake2_mac_hasher: VarBlake2b = VarBlake2b::new(CIPHER_SEED_MAC_BYTES)
-            .expect("Should be able to create blake2 hasher; will only panic if output size is 0 or greater than 64");
-        let mut hmac = [0u8; CIPHER_SEED_MAC_BYTES];
-        blake2_mac_hasher
-            .chain(plaintext.clone())
-            .chain([CIPHER_SEED_VERSION])
-            .chain(self.salt)
-            .chain(passphrase.as_bytes())
-            .finalize_variable(|res| hmac.copy_from_slice(res));
+        // generate the hash MAC
+        let mut hmac = Self::generate_hmac(
```

Comment:
For clarity I would rename this, since [HMAC](https://datatracker.ietf.org/doc/html/rfc2104) has a specific definition that isn't being used here. In this design the MAC is generated from a hash function, but it's not strictly HMAC.

Reply:
I rename it simply to `mac`.